### ISSUE TITLE
Try to get llvm+clang@14.0.6 to work on Toss4 Quartz

### ISSUE THREAD

**Issue by chapman39** [LLNL/serac#930]
Toss4 currently only has clang 14.0.6 installed, so when we build devtools we need to build llvm+clang@10.0.0 ourselves.

**Comment by samuelpmishLLNL** [comment]
potentially related: if we decide to start building llvm, configuring with the following flags will enable Enzyme support:

```
-DCMAKE_C_COMPILER=...
-DCMAKE_CXX_COMPILER=...
-DLLVM_TARGETS_TO_BUILD="host"
-DLLVM_ENABLE_PROJECTS=clang;ld
-DLLVM_ENABLE_PLUGINS=ON
-DLLVM_ENABLE_ASSERTIONS=ON
-DLLVM_ENABLE_RUNTIMES="libcxx;libcxxabi"
```

(although, I've only tested on LLVM11+, older versions of LLVM might require slightly different flags)

Also, including `clang-tools-extra` in the PROJECTS option will enable JiT compilation support in LLVM 15 or later.

**Comment by white238** [comment]
Fixed in #1090
